src/plugins/discover/public/application/apps/main/components/sidebar/discover_field_details.tsx

```diff
@@ -6,74 +6,31 @@
  * Side Public License, v 1.
  */
 
-import React, { useState, useEffect } from 'react';
-import { EuiIconTip, EuiText, EuiButton, EuiSpacer } from '@elastic/eui';
+import React from 'react';
+import { EuiText, EuiSpacer, EuiLink } from '@elastic/eui';
 import { FormattedMessage } from '@kbn/i18n/react';
-import { METRIC_TYPE, UiCounterMetricType } from '@kbn/analytics';
 import { DiscoverFieldBucket } from './discover_field_bucket';
-import { getWarnings } from './lib/get_warnings';
-import {
-  triggerVisualizeActions,
-  isFieldVisualizable,
-  getVisualizeHref,
-} from './lib/visualize_trigger_utils';
 import { Bucket, FieldDetails } from './types';
 import { IndexPatternField, IndexPattern } from '../../../../../../../data/public';
-import './discover_field_details.scss';
 
 interface DiscoverFieldDetailsProps {
   field: IndexPatternField;
   indexPattern: IndexPattern;
   details: FieldDetails;
   onAddFilter: (field: IndexPatternField | string, value: string, type: '+' | '-') => void;
-  trackUiMetric?: (metricType: UiCounterMetricType, eventName: string | string[]) => void;
 }
 
 export function DiscoverFieldDetails({
   field,
   indexPattern,
   details,
   onAddFilter,
-  trackUiMetric,
 }: DiscoverFieldDetailsProps) {
-  const warnings = getWarnings(field);
-  const [showVisualizeLink, setShowVisualizeLink] = useState<boolean>(false);
-  const [visualizeLink, setVisualizeLink] = useState<string>('');
-
-  useEffect(() => {
-    isFieldVisualizable(field, indexPattern.id, details.columns).then(
-      (flag) => {
-        setShowVisualizeLink(flag);
-        // get href only if Visualize button is enabled
-        getVisualizeHref(field, indexPattern.id, details.columns).then(
-          (uri) => {
-            if (uri) setVisualizeLink(uri);
-          },
-          () => {
-            setVisualizeLink('');
-          }
-        );
-      },
-      () => {
-        setShowVisualizeLink(false);
-      }
-    );
-  }, [field, indexPattern.id, details.columns]);
-
-  const handleVisualizeLinkClick = (event: React.MouseEvent<HTMLAnchorElement, MouseEvent>) => {
-    // regular link click. let the uiActions code handle the navigation and show popup if needed
-    event.preventDefault();
-    if (trackUiMetric) {
-      trackUiMetric(METRIC_TYPE.CLICK, 'visualize_link_click');
-    }
-    triggerVisualizeActions(field, indexPattern.id, details.columns);
-  };
-
   return (
     <>
-      <div className="dscFieldDetails">
```

Comment:
:information_source: This CSS class just added a couple of spacing (I replaced that by some well placed `<EuiSpacer />`) and a redundant text color. This allowed us to get rid of the whole SCSS for that component.